### PR TITLE
Josh dev

### PR DESCRIPTION
**Pull request by JoshuaWarejko**
- Join lobby as guess
- Some homepage styling

### CODE REVIEW

File: frontend/src/scss/_global.scss [hunk 1]
```diff
@@ -12,7 +12,7 @@
     }
 }
 
-h2 {
+h2, .h2 {
```
Comment (torch2424) [inline frontend/src/scss/_global.scss]: 🤔

File: frontend/src/app/providers/user-service/user.service.spec.ts [hunk 2]
```diff
@@ -1,15 +1,15 @@
 import { TestBed, inject } from '@angular/core/testing';
 
-import { UserServiceService } from './user-service.service';
+import { UserService } from './user.service';
```
Comment (torch2424) [inline frontend/src/app/providers/user-service/user.service.spec.ts]: 🔥

File: frontend/src/app/pages/home/home.html [hunk 3]
```diff
@@ -3,11 +3,11 @@
         <div class="overlay"></div>
         <div class="container has-text-centered">
             <div class="breathingRoom">
-                <h1 class="title">Paint With Bob</h1>
+                <h1 class="h2 title">Paint With Bob</h1>
```
Comment (torch2424) [inline frontend/src/app/pages/home/home.html]: 🤔 🤔 🤔

Comment (torch2424) [reply frontend/src/app/pages/home/home.html]: Seo purposes

File: frontend/src/scss/_global.scss (hunk 1 above)
Comment (JoshuaWarejko) [reply frontend/src/scss/_global.scss]: SEO 😉

**Pull request merged**